build.gradle

```diff
@@ -607,7 +616,10 @@ subprojects { subproject ->
     }
   }
 
-  check.dependsOn jacocoTestReport
+  check.dependsOn integration
+  integration.mustRunAfter test
+  jacocoTestReport.mustRunAfter test
+//  check.dependsOn jacocoTestReport
```

Comment:
Remove this commented out line if it is not needed.

Reply:
done